electron/generators/diory-generator.spec.js

```diff
@@ -1,188 +0,0 @@
-const { v4: uuid } = require('uuid')
```

Comment:
What do you think? Should we have unit test for generateDioryFromFile? What kind of?